testkit/src/main/scala/org/bitcoins/testkit/chain/ChainDbUnitTest.scala

```diff
@@ -0,0 +1,27 @@
+package org.bitcoins.testkit.chain
+
+import com.typesafe.config.ConfigFactory
+import org.bitcoins.chain.config.ChainAppConfig
+import org.bitcoins.testkit.BitcoinSTestAppConfig.ProjectType
+import org.bitcoins.testkit.{BitcoinSTestAppConfig, EmbeddedPg}
+
+trait ChainDbUnitTest extends ChainUnitTest with EmbeddedPg {
+
+  implicit override lazy val appConfig: ChainAppConfig = {
+    val memoryDb =
+      BitcoinSTestAppConfig.configWithEmbeddedDb(Some(ProjectType.Chain), pgUrl)
+    val chainConfig: ChainAppConfig =
+      BitcoinSTestAppConfig.getSpvTestConfig()
+    chainConfig.withOverrides(memoryDb)
+  }
+
+  override lazy val mainnetAppConfig: ChainAppConfig = {
```

Comment:
How do we ensure that we avoid messing with people's real mainnet data in tests?

Reply:
We still use a `tmpDir()` inside of the `BitcoinSAppConfig()`: https://github.com/bitcoin-s/bitcoin-s/blob/53f398fe45229ad63116fd20a0b6073957f83c54/testkit/src/main/scala/org/bitcoins/testkit/BitcoinSTestAppConfig.scala#L33